### ISSUE TITLE
Rule NS2003 does not detect InternalsVisibleTo correctly

### ISSUE THREAD

**Issue by meziantou** [nsubstitute/NSubstitute.Analyzers#54]
I do have the attribute `InternalsVisibleTo` in the file `AssemblyInfo.cs`

````csharp
[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
````

However, the analyser adds a warning and suggests a fix when I use `Substitute.ForPartsOf<MyInternalType>()`. The fix add the attribute in the file that contains the class MyInternalType. But this is not needed as the attribute is already declared in another file.

Expected behavior: it should not report a warning if there is `[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]` somewhere in the project.

**Comment by tpodolak** [comment]
@meziantou is the InternalsVisibleTo defined in AssemblyInfo.cs  in the same project as MyInternalType class?

**Comment by meziantou** [comment]
It is. The code compiles and the tests passes, so I assume the way it is declared is ok. Here's the structure of my solution:

````
project.csproj
  - MyInternalType.cs
  - AssemblyInfo.cs (contains InternalsVisibleTo("DynamicProxyGenAssembly2"))
project.tests.csproj
  - Tests.cs (contains Substitute.ForPartsOf<MyInternalType>)
````

**Comment by tpodolak** [comment]
Confirm, this is a bug, will fix it over the weekend - maybe sooner. Thanks for reporting

**Comment by tpodolak** [comment]
@meziantou, fix released in [1.0.3](https://www.nuget.org/packages/NSubstitute.Analyzers.CSharp/1.0.3)
